app/renderer/components/tabs/tab.js

```diff
@@ -337,34 +353,39 @@ class Tab extends React.Component {
 }
 
 const styles = StyleSheet.create({
-  // Windows specific style
-  tabForWindows: {
-    color: '#555'
-  },
-
   tab: {
     borderWidth: '0 1px 0 0',
     borderStyle: 'solid',
     borderColor: '#bbb',
     boxSizing: 'border-box',
-    color: '#5a5a5a',
+    color: theme.tab.color,
```

Comment:
On theme.js `theme.tab.color` is defined as `#000`. Is this change expected?

Reply:
you are right